### PR TITLE
[CDAP-13535] Restyle top nav

### PR DESCRIPTION
**Pull request by ajainarayanan**
- Changes the top nav styling + links in it.
- Changes CDAP logo (removes Cask Logo)
- Changes default cdap-orange to gcp-blue
- Changes all reference of cdap-orange to gcp-blue
- Fixes About CDAP modal footer
- Removes SplashScreen component (yay!)
- Fixes all references of Cask Market to Hub

Build: https://builds.cask.co/browse/CDAP-UDUT20
JIRA: https://issues.cask.co/browse/CDAP-13535

### CODE REVIEW

**Comment by bdmogal** [comment]
@ajainarayanan will this PR address https://issues.cask.co/browse/CDAP-13667 and https://issues.cask.co/browse/CDAP-13668. If not, can it?

**Comment by ajainarayanan** [comment]
@bdmogal sure will do.

**Comment by tonybach** [comment]
After a quick search I'm still seeing two references to `$cdap-orange` in the `PipelineConfigurations.scss` and `PipelineModeless.scss`

**Comment by tonybach** [comment]
`$orange-01` is also `#ff6600`, not sure if we want to change those references as well?

**Comment by tonybach** [comment]
I'm seeing a blank dropdown when clicking on 'Local Sandbox' from the Angular apps (Pipelines & Tracker)

image

File: cdap-ui/app/styles/variables.less [hunk 1]
```diff
@@ -159,3 +159,5 @@
 @green-01: #01b133;
 @green-02: #3cc801;
 @green-03: #8af302;
+
+@gcp-blue: #3b78e7;
```
Comment (tonybach) [inline cdap-ui/app/styles/variables.less]: nit: need new line

**Comment by ajainarayanan** [comment]
There is still usage of `orange-02` to `orange-04` in our app. I will let @leacd suggest corresponding gcp blue shades as a 1-to-1 replacement

**Comment by nitinmotgi** [comment]
Rename gcp-blue . No product names please.

File: cdap-ui/app/styles/variables.less [hunk 2]
```diff
@@ -159,3 +159,5 @@
 @green-01: #01b133;
 @green-02: #3cc801;
 @green-03: #8af302;
+
+@brand-primary-color: #3b78e7;
```
Comment (elfenheart) [inline cdap-ui/app/styles/variables.less]: nit: new line

**Comment by tonybach** [comment]
I noticed that in the Angular apps, the 'Local Sandbox' dropdown doesn't show the border separating the options.

image

image

File: cdap-ui/app/login/components/Footer/index.js [hunk 3]
```diff
@@ -1,48 +0,0 @@
...
-        <div className="container">
-          <div className="row text-muted">
-            <div className="text-uppercase">
-              <p className="text-xs-center">
-                <span>Copyright &copy; {this.copyrightYear} Cask Data, Inc.</span>
-              </p>
-            </div>
-          </div>
-        </div>
-      </footer>
```
Comment (elfenheart) [inline cdap-ui/app/login/components/Footer/index.js:41]: I think you need to change the reference to Footer component in login. I can't build
```
[WEBPACK] Errors building ./login.js
Module not found: Error: Can't resolve './components/Footer' in '/Users/edwinelia/source/cdap/cdap-ui/app/login'
```

Comment (ajainarayanan) [reply cdap-ui/app/login/components/Footer/index.js:41]: argh! sorry about that. Thought we were already referencing the footer from cdap. Didn't realize the relative path.

**Comment by elfenheart** [comment]
screen shot 2018-07-11 at 11 31 32 am

The CDAP image is white on white. 


Other than that, things seem to work

**Comment by elfenheart** [comment]
👍 code LGTM. Things seemed to work

**Comment by tonybach** [comment]
Code LGTM 👍

**Comment by leacd** [comment]
LGTM

**Pull request merged**